app/services/no_bp_measure_service.rb

```diff
@@ -18,6 +18,9 @@ def call
   end
 
   def visited_without_bp_taken_for(period)
```

Comment:
Could we call this method something like `visited_without_bp_taken_count` since it's only a count and not the actual patients?

Reply:
👍